packit/cli/utils.py

```diff
@@ -104,81 +104,107 @@ def get_packit_api(
         try_local_dir_last=True,
         package_config_path=config.package_config_path,
     )
-    logger.debug(f"job_config_index: {job_config_index}")
-    if job_config_index is not None and isinstance(package_config, PackageConfig):
-        if job_config_index >= len(package_config.jobs):
-            raise PackitException(
-                "job_config_index is bigger than number of jobs in package config!"
+
+    packages_configs = [package_config]
+    multiple_packages = False
+    try:
+        package_config.upstream_project_url
+    except AttributeError as e:
+        if "there is more than one package in the config" in str(e):
+            packages_configs = [
+                package_config.packages[pkg] for pkg in package_config.packages.keys()
+            ]
+            multiple_packages = True
+
+    packit_api_collection = []
+    for package_config in packages_configs:
+
+        logger.debug(f"job_config_index: {job_config_index}")
+        if job_config_index is not None and isinstance(package_config, PackageConfig):
+            if job_config_index >= len(package_config.jobs):
```

Comment:
It seems to me, that this won't work when there are multiple packages defined: a `package_config` taken from `package_config.packages` is going to be a `CommonPackageConfig` and won't have a `jobs` attribute.

Reply:
You are right, I missed it. But I think we can create a new object, let me call it `PackageConfigView` which has all the jobs and attributes used by a package and taken from a  `MultiplePackages` config.

I made the same mistake also in the new PR: https://github.com/packit/packit/pull/1840, but I think it easy to fix.